stopcovid/trigger_reminders/trigger_reminders.py

```diff
@@ -0,0 +1,68 @@
+from typing import List
+import json
+import boto3
+import os
+
+from stopcovid.status.drill_instances import DrillInstance, DrillInstanceRepository
+
+from . import persistence
+
+REMINDER_TRIGGER_FLOOR_MINUTES = 60 * 4
+REMINDER_TRIGGER_CEIL_MINUTES = 60 * 12
```

Comment:
per my other comment about daytime hours only, make sure that the ceiling is high enough that we wouldn't miss a reminder after a night has elapsed.

Assuming send times from 1-8 pm eastern time, we probably want to do 24 hours.